src/main.cpp

```diff
@@ -583,4 +602,13 @@ void loop()
     
     processStrips(pwm);
   }
+
+  // Publish sensor telemetry (if any)
+  StaticJsonDocument<150> telemetry;
```

Comment:
Doing the telemetry this way means the sensor lib doesn't need any `OXRS_MQTT` dependency and makes it easier to use, since now we have the different OXRS platform libs, we no longer have direct access to the MQTT client.